apps/notifications/models.py

```diff
@@ -0,0 +1,161 @@
+import logging
+
+from django.conf import settings
+from django.db import models
+
+from apps.authentication.models import OnlineUser as User
+from apps.gallery.models import ResponsiveImage
+
+from .constants import PermissionType
+
+logger = logging.getLogger(__name__)
+
+
+class Notification(models.Model):
+    """
+    Standard notification message which is sent to a single user on creation
+    """
+
+    recipient = models.ForeignKey(
+        to=User,
+        verbose_name="Mottaker",
+        related_name="notifications",
+        on_delete=models.CASCADE,
+    )
+    created_date = models.DateTimeField(auto_now_add=True)
+    sent_email = models.BooleanField(default=False)
+    sent_push = models.BooleanField(default=False)
+    permission = models.ForeignKey(
+        to="Permission", related_name="notifications", on_delete=models.DO_NOTHING
+    )
+    from_email = models.EmailField(default=settings.DEFAULT_FROM_EMAIL)
+
+    image = models.ForeignKey(
+        to=ResponsiveImage,
+        related_name="notifications",
+        blank=True,
+        null=True,
+        on_delete=models.SET_NULL,
+    )
+
+    title = models.CharField(max_length=60)
+    body = models.CharField(max_length=512)
+    tag = models.CharField(max_length=50, null=True, blank=True)
```

Comment:
This was done so that the serialized value would result in null instead of an empty string. Tags are used as an identifier for notifications, which means that when you have a notification on your device, and you receive a new one with the same tag, then the previous one will be dismissed. Having an empty string will make all tags without a specified tag dismiss any previous one

Reply:
Okay, but wouldn't it be more reasonable to instead make the serializer emit `null`, when the string is empty instead?

Reply:
Commented it